mysql/create.sql

```diff
@@ -0,0 +1,104 @@
+CREATE DATABASE mintern;
```

Comment:
It's not specified, but I think it is safe to assume databases have a similar precedence than tables, therefore you should use UpperCamelCase for the name.

Reply:
done